### PR TITLE
Crystal

### PR DESCRIPTION
**Pull request by itscrystalm**

### CODE REVIEW

File: Library_group9/libraryAutomation_group9.5/libraryAutomation_group9.5.iml [hunk 1]
```diff
@@ -1,66 +1,66 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (akilahrabb) [inline Library_group9/libraryAutomation_group9.5/libraryAutomation_group9.5.iml:1]: please add .iml file to gitignore, along with .idea and target folder, then,,,
Clean your files up by running Maven clean.
Then, run cukesrunner again to generate a new clean target folder.
Update all branches and merge them into one another.
Push an amended commit to your remote.

File: Library_group9/libraryAutomation_group9.5/src/test/java/com/library/pages/LibraryLogout.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
+package com.library.pages;
```
Comment (akilahrabb) [inline Library_group9/libraryAutomation_group9.5/src/test/java/com/library/pages/LibraryLogout.java:1]: Add https://github.com/akilahrabb/Library_Group9/blob/main/src/test/java/com/library/pages/LibraryLoginPage.java

File: Library_group9/libraryAutomation_group9.5/src/test/java/com/library/runners/CukesRunner.java [hunk 3]
```diff
@@ -14,7 +14,7 @@
         features = "src/test/resources/features",
         glue = "com/library/step_definitions",
         dryRun = false,
-        tags = ""
```
Comment (akilahrabb) [inline Library_group9/libraryAutomation_group9.5/src/test/java/com/library/runners/CukesRunner.java:17]: Replace with https://github.com/akilahrabb/Library_Group9/blob/main/src/test/java/com/library/runners/CukesRunner.java

File: Library_group9/libraryAutomation_group9.5/src/test/java/com/library/step_definitions/Logout_StepDefinitions.java [hunk 4]
```diff
@@ -0,0 +1,80 @@
+package com.library.step_definitions;
```
Comment (akilahrabb) [inline Library_group9/libraryAutomation_group9.5/src/test/java/com/library/step_definitions/Logout_StepDefinitions.java:1]: Add, but your file structure must be updated.
https://github.com/akilahrabb/Library_Group9/pull/10/files?file-filters%5B%5D=.class&file-filters%5B%5D=.feature&file-filters%5B%5D=.html&file-filters%5B%5D=.java&file-filters%5B%5D=.json#diff-62bc781f9f21449de3cb1cef884d72bc14d128e6f2031d919cdc25695f1994f6

File: Library_group9/libraryAutomation_group9.5/src/test/resources/features/login.feature [hunk 5]
```diff
@@ -1,5 +1,6 @@
 Feature: As a user, I should be able to login to the library app
```
Comment (akilahrabb) [inline Library_group9/libraryAutomation_group9.5/src/test/resources/features/login.feature:2]: Add the updated version 

https://github.com/akilahrabb/Library_Group9/blob/main/src/test/resources/features/Login_1.feature

File: Library_group9/libraryAutomation_group9.5/target/cucumber-report.html [hunk 6]
```diff
@@ -299,7 +299,7 @@
...
...4bf870","testCaseStartedId":"182a8294-a932-4b4b-87f9-e8795cff9bdf"}},{"testStepStarted":{"timestamp":{"seconds":"1614182105","nanos":508000000},"testStepId":"78425044-3f56-43f6-b743-f787ec55044f","testCaseStartedId":"182a8294-a932-4b4b-87f9-e8795cff9bdf"}},{"testStepFinished":{"testStepResult":{"status":"PASSED","duration":{"nanos":1000000}},"timestamp":{"seconds":"1614182105","nanos":509000000},"testStepId":"78425044-3f56-43f6-b743-f787ec55044f","testCaseStartedId":"182a8294-a932-4b4b-87f9-e8795cff9bdf"}},{"testStepStarted":{"timestamp":{"seconds":"1614182105","nanos":510000000},"testStepId":"aa4500ea-af09-42c4-9532-24ba1781306c","testCaseStartedId":"182a8294-a932-4b4b-87f9-e8795cff9bdf"}},{"testStepFinished":{"testStepResult":{"status":"PASSED","duration":{"seconds":"6","nanos":27000000}},"timestamp":{"seconds":"1614182111","nanos":537000000},"testStepId":"aa4500ea-af09-42c4-9532-24ba1781306c","testCaseStartedId":"182a8294-a932-4b4b-87f9-e8795cff9bdf"}},{"testCaseFinished":{"timestamp":{"seconds":"1614182111","nanos":538000000},"testCaseStartedId":"182a8294-a932-4b4b-87f9-e8795cff9bdf"}},{"testRunFinished":{"success":true,"timestamp":{"seconds":"1614182111","nanos":543000000}}}];
```
Comment (akilahrabb) [inline Library_group9/libraryAutomation_group9.5/target/cucumber-report.html:302]: Remove this file, and add to git ignore
https://github.com/akilahrabb/Library_Group9/pull/10/files?file-filters%5B%5D=.class&file-filters%5B%5D=.feature&file-filters%5B%5D=.html&file-filters%5B%5D=.java&file-filters%5B%5D=.json#diff-3c7a47aec2d5a13dceb0e726546a7279f1efc639f4a4918580432ac670b5f153arget folder to gitignore

File: Library_group9/libraryAutomation_group9.5/target/test-classes/features/logout.feature [hunk 7]
```diff
@@ -0,0 +1,14 @@
+Feature: User should be able to log out of library app.
```
Comment (akilahrabb) [inline Library_group9/libraryAutomation_group9.5/target/test-classes/features/logout.feature:1]: add but correct file structur
https://github.com/akilahrabb/Library_Group9/pull/10/files?file-filters%5B%5D=.class&file-filters%5B%5D=.feature&file-filters%5B%5D=.html&file-filters%5B%5D=.java&file-filters%5B%5D=.json#diff-078080a6dbdff1aafe5a07fb4b40141d7006f7db06f81e8e6654cc3f3100d851e

**Review by akilahrabb — CHANGES_REQUESTED** [review]

**Review by itscrystalm — COMMENTED** [review]
ok, thank you

**Pull request closed without merging**